### ISSUE TITLE
[FEATURE REQUEST] Add a "report bug" / "give feedback" button

### ISSUE THREAD

**Issue by JanTuts** [prusa3d/Prusa-Link-Web#357]
I think it would be a good idea to add a "report bug" / "give feedback" button somewhere on the interface, directly linking to this github, specifically because it's _quite confusing_ as to which "Prusa Link" is actually which. 

I was about to post an unrelated feature request over at [https://github.com/prusa3d/Prusa-Link](https://github.com/prusa3d/Prusa-Link), before I realised that "Prusa Link" (as the logo on the interface calls itself) is not in fact "Prusa Link", but "Prusa Link Web".

(The fact that [the instructions on how to use Prusa Link Web with the Prusa Mini](https://help.prusa3d.com/article/prusa-connect-local-mini_133279) still calls it "Prusa Connect Local" also isn't helping.)

**Comment by mix579** [comment]
Makes sense. Someone should really sit down and think through how you guys label these different components of the PrusaLink/PrusaConnect infrastructure. 

We have PrusaConnect, PrusaLink for the Mk3S systems, Prusalink baked into the Mini firmware, and the PrusaLink Web interfaces, which are DIFFERENT for the Mk3S and the Mini . How any of those pieces correlate with the different github repositories is less than obvious. Clarifying both the terms used for the infrastructure components *and* the github repos would be very helpful.

**Comment by Prusa-Support** [comment]
This sounds like a good idea. Thanks for it.

[Prusa Connect](connect.prusa3d.com) already has a _share your feedback_ bar (like [Printables](https://www.printables.com/)).
It is not connected to GitHub though: GitHub may not be very user-friendly for regular customers indeed.

Prusa Link is not a powerful platform as it is hosted by the printer.
Prusa Link is the most basic interface and a bridge between the printer and Prusa Connect.
Prusa Connect is hosted on our servers instead, giving the users full remote access and control to all connected printers. More (and more advanced) features will be available in Prusa Connect for all Prusa printers, reducing the need for the users to directly open the printer's IP address on the browser.

This said, our developers may want to keep Prusa Link UI and functionality as basic as possible for now, but we will keep this issue open as a feature request for the time being.

Michele Moramarco
Prusa Research
